### ISSUE TITLE
Reviewer notes inside of tables are generating corrupted DOC outputs

### ISSUE THREAD

**Issue by manuelfuenmayor** [metanorma/isodoc#471]
From IHO-102: https://github.com/metanorma/S-102-Product-Specification/pull/11

The reviewer notes inside of this table generate a corrupted DOC output that cannot be oppened:

```adoc
[[tab-attributes-of-bathymetrycoverage-feature-container-group]]
[cols="<,<,<,^,<,<"]
.Attributes of *BathymetryCoverage* feature container group
|===
| No
| Name
| Camel Case
| Mult
| Data Type
| Remarks

| 1
| Data organization index
| dataCodingFormat
| 1
| Enumeration
| Value: 9

| 2
| Dimension
| dimension
| 1
| Integer unsigned +
8-bit
| Value: 2

| 3
| Common point rule
| commonPointRule
| 1
a| Enumeration

[reviewer=Lawrence Haynes Haselmaier]
****
If we do not adopt BSH position, "unsigned 16-bit" must be added after "Enumeration"
****

| Value: 1 (average) or other values from <<iho-s100,table=10c-20>>.

| 4
| Horizontal position uncertainty
a| horizontalPositionUncertainty

[reviewer=Lawrence Haynes Haselmaier]
****
After Andy/Anthony/Lawrence meeting: This is likely to vary throughout an S-102 surface. What is the value expected here? Is it the maximum? How does this relate to the nodal uncertainty values?

Chair has action to discuss with PT at-large
****

| 1
| Float 32-bit
| Value: -1.0 (if unknown or not available)

| 5
| Vertical position uncertainty
a| verticalUncertainty

[reviewer=Lawrence Haynes Haselmaier]
****
After Andy/Anthony/Lawrence meeting: This is likely to vary throughout an S-102 surface. What is the value expected here? Is it the maximum? How does this relate to the nodal uncertainty values?

Chair has action to discuss with PT at-large
****

| 1
| Float 32-bit
| Value: -1.0 (if unknown or not available)

| 6
| Number of feature instances
| numInstances
| 1
| Integer unsigned +
8-bit
| Value: 1

| 7a
.2+| Sequencing rule
| sequencingRule.type
^| 1
| Enumeration
| Value: 1 (linear) see <<iho-s100,table=10c-21>>

| 7b
| sequencingRule.scanDirection
^| 1
| String
| Value: <axisNames entry> (comma-separated). For example, "latitude,longitude". Reverse scan direction along an axis is indicated by prefixing a '-' sign to the axis name. See <<scanDirection>>

| 8
| Interpolation type
| interpolationType
| 1
| Enumeration
| Code value from <<iho-s100,table=10c-22>>

|===
```

**Comment by anermina** [comment]
I believe the initial markup that I used for adding the reviewer's comments is wrong and shouldn't be used. As shown in Manuel's example, a standalone comment was added to the table, which doesn't make sense, because standalone comments are rendered at the end of the section (if specified) - they are not rendered where they are placed. Moreover, these should not be standalone, but rather ranged comments.

Markup has been updated in https://github.com/metanorma/S-102-Product-Specification/pull/11/commits/8c1ebf365fd1b4078a49d68d4b353f91ec5f2f6a, so I'm closing this issue.
